### DISCUSSION TITLE
Support PhotoSwipe for inline images within paragraphs

### DISCUSSION THREAD

**Discussion by yuna0x0** [CaiJimmy/hugo-theme-stack#809] [Ideas]
> I knew that this theme is currently not accepting new feature requests.
> Just giving some thoughts here for the future iteration (if any). (๑˃ᴗ˂)ﻭ

Currently, inline images within paragraphs don't have PhotoSwipe support.

Having a shortcode that adds these images into PhotoSwipe, or a frontmatter that adds every image on the page into PhotoSwipe would be nice.
Otherwise, if a user wants to zoom in on the image in the article, they will have to open it in a new tab.

**Comment by CaiJimmy** [comment]
Sorry for the late reply.

If you want inline images to work with PhotoSwipe too, you can override `gallery.ts`, and comment out this line which does that check:

https://github.com/CaiJimmy/hugo-theme-stack/blob/c6d1d73dd21ff5e1c9c52db358e49857b3e0a6df/assets/ts/gallery.ts#L79C12-L79C43
